documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -437,16 +437,28 @@ For instance you may leave this to the default (`false`) during testing to ident
 and set it to `true` in production environments to fall back to index uninverting for uncovered sorts accidentally left over.
 
 [[id-annotation]]
-===== @Id
+===== Document identifier: @DocumentId or @Id
 
-Finally, the id property of an entity is a special property used by Hibernate Search to ensure index
-unicity of a given entity. By design, an id has to be stored and must not be tokenized. It is also
-always string encoded, even if the id is a number. To mark a property as index id, use the
-`@DocumentId` annotation. If you are using JPA and you are using `@Id` you can omit `@DocumentId`.
-The chosen entity id will also be used as document id.
+Finally, the document identifier is a special field used by Hibernate Search to ensure
+there will only ever be one document per entity in the index,
+and to retrieve documents when they should be deleted.
```

Comment:
s/retrieve/identify/ ?

I wouldn't suggest we actually "retrieve" a document to delete it